README.md

```diff
@@ -36,13 +36,14 @@ cp .env.template .env
 2. Start Metro
 ```
 yarn start
+# add --reset-cache to wipe metro cache (required for .env changes)
 ```
-
 3. Start the app
 ```
 cd ios
 pod install
 yarn ios
+# add --device 'iPhone' (replacing iPhone with your device name) to run on device. add --simulator 'iPhone 14' to specify iPhone 14 simulator
```

Comment:
should we mention this can also be done with xcode?

Reply:
done